config/default.json

```diff
@@ -13,6 +13,10 @@
     "CONNECT_TIMEOUT": 45000,
     "RECONNECT_INTERVAL": 5000
   },
+  "DB_CONNECTION": {
+    "DB_CONNECTION_POOL_MIN": 10,
+    "DB_CONNECTION_POOL_MAX": 30
```

Comment:
You should remove the `DB_CONNECTION_` prefix portion for `POOL_MAX` as its already a sub-object

Reply:
Makes logical sense - have updated.